substrate-archive/src/error.rs

```diff
@@ -13,18 +13,21 @@
 // You should have received a copy of the GNU General Public License
 // along with substrate-archive.  If not, see <http://www.gnu.org/licenses/>.
 
-use std::{env, fmt, io};
+use std::{env, fmt, io, num};
 use thiserror::Error;
 
 pub type Result<T, E = ArchiveError> = std::result::Result<T, E>;
 
 /// Substrate Archive Error Enum
 #[derive(Debug, Error)]
 pub enum ArchiveError {
+	// Rust std error
```

Comment:
I fine with removing the comment or fixing the comment

```suggestion
	// Rust std io error
```